pages/about.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: page
+title: About Solid
+permalink: /about
+
+---
+
+# About Solid
+
+Solid is a mid-course correction for the Web by its creator, 
+[Sir Tim Berners-Lee](/team). It realizes Tim's original vision for the 
+Web as a medium for the secure, decentralized exchange of public 
+and private data.
+
+<img src="/assets/img/timbl-cern.jpg"/>
+
+## Why fix the Web?
+
+The [first web browser](https://worldwideweb.cern.ch/) was also an editor. 
+The idea being that not only could everyone read content on the web, but 
+they could also help create it. It was 
+to be a collaborative space for all mankind.
+
+However, when the first browser that popularized the web came along, called 
+Mosaic, it included multimedia and editing was taken out. It was considered 
+too difficult a problem. This change was the first curtailing of the web’s 
+promise and spawned an effort led by Tim and others to get the write 
+functionality back. It was dubbed the ‘read-write web’ and led to Richard 
+McManus' 
+[seminal article](https://web.archive.org/web/20181214015324/http://readwrite.com/2003/04/19/the_readwrite_w) 
+published in 2003.
+
+The issue with writing data, as Wikipedia and others have learned, is that you 
+need a degree of control over who can write what. That means you need to have 
+permissions to dictate what individuals can do to the data. And to have 
+permissions you need to have a system for identity - a way of uniquely 
+authenticating that an individual is who they purport to be.
+
+Of course the social networks solved this problem within their own systems 
+using their own specific identity and access control, but these were not 
+standard, not interoperable, and gave you no choice in what applications you
+could use to access that data. You had to have your entire personal
+or professional life within one silo for it to work. And since the Web
+is ubiquitous, these silos exist across the data spectrum, from social
+and medical to financial and civil. 
+
+When your data is siloed away from you:
+
+* You have little control over how it is used, or who is using it. 
+* You have little choice in which applications you can use to access it.
+* It is hard to use as a cohesive unit, specifically because it is siloed,
+scattered across proprietary vendors, interfaces, and data formats.
+
```

Comment:
Add something about you not knowing what data an organisation has about you, or not understanding that data?